render-tests/raster-extent/maxzoom/style.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Having an `expected.png` that is an all-white png makes me nervous that we could mistake "no data" for a passing test. I would prefer to see some output, even if just a solid background color.